packages/client/src/hooks/useAccount.js

```diff
@@ -1,23 +1,28 @@
 import { REGULAR_LIMIT } from 'constants/constants';
-import { COIN_TYPES } from 'constants/enums';
 import { COIN_TYPE_TO_META } from 'constants/maps';
 import { mutate, query, tx } from '@onflow/fcl';
-import { GetAccountBalanceByContractName, SEND_TOKENS_TO_TREASURY } from 'flow';
+import {
+  GET_ACCOUNT_BALANCE,
+  SEND_TOKENS_TO_TREASURY,
+  VAULT_EXIST_CHECK,
+} from 'flow';
 
-const COIN_TYPE_LIST = [COIN_TYPES.FLOW, COIN_TYPES.FUSD, COIN_TYPES.USDC];
+const COIN_TYPE_LIST = Object.keys(COIN_TYPE_TO_META);
```

Comment:
if it's now entire list maybe this line can be removed and directly use it inside method.

Reply:
I would leave it like this if that is okay, because this variable is used in a couple of places so might be easier to read the code